### DISCUSSION TITLE
Works with Azure log analytics workspace?

### DISCUSSION THREAD

**Discussion by Gbl13** [grafana/grafana-infinity-datasource#281] [Q&A] [answered]
Hello!!
I Cannot find the grafana-azure-datasource plugin to install on my Grafana, seems like this have been excluded, can I use the Infinity plugin to integrate my Azure log analytics workspace with Grafana?

**Comment by yesoreyeram** [comment, marked as answer]
Yes. With the recent support to oauth client credentials in the plugin,   you can connect to azure monitor / log analytics queries.

**Reply by yesoreyeram** [reply]
I added https://github.com/yesoreyeram/grafana-infinity-datasource/discussions/404 for detailed instruction on how to setup azure authentication. Hope it helps

**Comment by yesoreyeram** [comment]
Btw, what do you mean by you can't find grafana-azure-datasource . Azure is one of grafana's native plugin right?
